contracts/common/MiniMeToken.sol

```diff
@@ -359,11 +361,11 @@ contract MiniMeToken is Controlled {
         bool _transfersEnabled
     ) public returns(address)
     {
-        if (_snapshotBlock == 0)
-            _snapshotBlock = block.number;
+        uint256 snapshot = _snapshotBlock == 0 ? block.number - 1 : _snapshotBlock;
```

Comment:
Was there a reason for changing this to be `block.number - 1` rather than `block.number` when `_snapshotBlock == 0`?

Reply:
Yes, there is a small race condition that can happen if using the same block number. We don't use it anywhere but fixed it jic